internal/npm_install/npm_install.bzl

```diff
@@ -267,6 +268,7 @@ cd "{root}" && "{npm}" {npm_args}
         [repository_ctx.path("npm.cmd" if is_windows_host else "npm")],
         timeout = repository_ctx.attr.timeout,
         quiet = repository_ctx.attr.quiet,
+        environment = {"build_bazel_rules_nodejs_version": VERSION},
```

Comment:
I think all caps for an environment variable: `BUILD_BAZEL_RULES_NODEJS_VERSION`